notebooks/integrations/integration-sklearn.ipynb

```diff
@@ -239,21 +249,132 @@
+      "  warnings.warn(\n",
+      "/opt/miniconda3/envs/rubicon-ml-dev/lib/python3.10/site-packages/sklearn/linear_model/_stochastic_gradient.py:705: ConvergenceWarning: Maximum number of iteration reached before convergence. Consider increasing max_iter to improve the fit.\n",
+      "  warnings.warn(\n",
+      "/opt/miniconda3/envs/rubicon-ml-dev/lib/python3.10/site-packages/sklearn/linear_model/_stochastic_gradient.py:705: ConvergenceWarning: Maximum number of iteration reached before convergence. Consider increasing max_iter to improve the fit.\n",
+      "  warnings.warn(\n",
+      "/opt/miniconda3/envs/rubicon-ml-dev/lib/python3.10/site-packages/sklearn/linear_model/_stochastic_gradient.py:705: ConvergenceWarning: Maximum number of iteration reached before convergence. Consider increasing max_iter to improve the fit.\n",
+      "  warnings.warn(\n",
+      "/opt/miniconda3/envs/rubicon-ml-dev/lib/python3.10/site-packages/sklearn/linear_model/_stochastic_gradient.py:705: ConvergenceWarning: Maximum number of iteration reached before convergence. Consider increasing max_iter to improve the fit.\n",
+      "  warnings.warn(\n",
+      "/opt/miniconda3/envs/rubicon-ml-dev/lib/python3.10/site-packages/sklearn/linear_model/_stochastic_gradient.py:705: ConvergenceWarning: Maximum number of iteration reached before convergence. Consider increasing max_iter to improve the fit.\n",
+      "  warnings.warn(\n",
+      "/opt/miniconda3/envs/rubicon-ml-dev/lib/python3.10/site-packages/sklearn/linear_model/_stochastic_gradient.py:705: ConvergenceWarning: Maximum number of iteration reached before convergence. Consider increasing max_iter to improve the fit.\n",
+      "  warnings.warn(\n",
+      "/opt/miniconda3/envs/rubicon-ml-dev/lib/python3.10/site-packages/sklearn/linear_model/_stochastic_gradient.py:705: ConvergenceWarning: Maximum number of iteration reached before convergence. Consider increasing max_iter to improve the fit.\n",
+      "  warnings.warn(\n",
+      "/opt/miniconda3/envs/rubicon-ml-dev/lib/python3.10/site-packages/sklearn/linear_model/_stochastic_gradient.py:705: ConvergenceWarning: Maximum number of iteration reached before convergence. Consider increasing max_iter to improve the fit.\n",
+      "  warnings.warn(\n",
+      "/opt/miniconda3/envs/rubicon-ml-dev/lib/python3.10/site-packages/sklearn/linear_model/_stochastic_gradient.py:705: ConvergenceWarning: Maximum number of iteration reached before convergence. Consider increasing max_iter to improve the fit.\n",
+      "  warnings.warn(\n",
+      "/opt/miniconda3/envs/rubicon-ml-dev/lib/python3.10/site-packages/sklearn/linear_model/_stochastic_gradient.py:705: ConvergenceWarning: Maximum number of iteration reached before convergence. Consider increasing max_iter to improve the fit.\n",
+      "  warnings.warn(\n"
+     ]
+    },
```

Comment:
I also delete long lists of warnings like this so they don't clutter the examples

```suggestion
```